tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/test_formula_8_14.py

```diff
@@ -52,3 +52,25 @@ def test_raise_error_if_n_b_is_negative(self) -> None:
 
         with pytest.raises(NegativeValueError):
             Form8Dot14EquivalentDiameterBundledBars(diameter=diameter, n_b=n_b)
+
+    @pytest.mark.parametrize(
+        ("representation", "expected"),
+        [
+            (
+                "complete",
+                (r"\Phi_n = \min \left(55, \Phi \cdot \sqrt{n_b} \right) = \min \left(55, 16.00 \cdot \sqrt{4.00} \right) = 32.00"),
```

Comment:
![afbeelding](https://github.com/Blueprints-org/blueprints/assets/44614728/52253add-c257-4cf1-88e5-272a5d212a3e)
```suggestion
                (r"Ø_n = \min \left(55 \ \text{mm}, Ø \cdot \sqrt{n_b} \right) = \min \left(55 \ \text{mm}, 16.00 \cdot \sqrt{4.00} \right) = 32.00"),
```